src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1400,6 +1400,7 @@ private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
 
         protected virtual void EnterParameter(ParameterSymbol parameter)
         {
+            Visit(parameter.ExplicitDefaultExpression);
```

Comment:
> why would an abstract method be particularly interesting here?

For example, see a behavior change in CS1736ERR_DefaultValueMustBeConstant01 (SymbolErrorTests.cs). I would expect the same behavior change with an abstract method.